src/solvers/strings/string_constraint_generator.h

```diff
@@ -111,6 +111,12 @@ std::pair<exprt, string_constraintst> add_axioms_for_string_of_int_with_radix(
   const namespacet &ns,
   array_poolt &array_pool);
 
+/// Given an exprt and a string representing a character range of the form
+/// [low_char]-[high_char], return an and_exprt that represents restricting
+/// the selection of the character to the range.
+and_exprt
```

Comment:
:pick: there is no particular reason this should return an `and_exprt`, I suggest making it return an `exprt`